saleor/core/error_codes.py

```diff
@@ -22,3 +22,9 @@ class TranslationErrorCode(Enum):
     GRAPHQL_ERROR = "graphql_error"
     NOT_FOUND = "not_found"
     REQUIRED = "required"
+
+
+class UploadErrorCode(Enum):
+    GRAPHQL_ERROR = "graphql_error"
+    NOT_FOUND = "not_found"
```

Comment:
Is there a case when the `NOT_FOUND` error code would be returned for this mutation? Same for `REQUIRED`, since we don't have any validation that would use it. Fields are required at the schema level.

Reply:
No, I dropped those error codes.